assets/scripts/sentiment/SentimentSurveyContainer.jsx

```diff
@@ -36,7 +36,9 @@ function SentimentSurveyContainer (props) {
         setVisible(true)
       }, SURVEY_DELAY_BEFORE_APPEAR)
     }
-  })
+    // only re-run the effect if the value of isDismissed has changed
```

Comment:
This took me a minute to get to, more info here: 
https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects